src/server/bin/kibana.bat

```diff
@@ -9,6 +9,7 @@ set NODE=%DIR%\node\node.exe
 set SERVER=%DIR%\src\bin\kibana.js
 set NODE_ENV="production"
 set CONFIG_PATH=%DIR%\config\kibana.yml
+set NPM=npm
```

Comment:
I applied this change, however the path needed to be slightly different for me. Please review and let me know if that path will work for all instances, or if that is a windows-specific nuance.